docs/querying/sql-functions.md

```diff
@@ -28,17 +28,39 @@ sidebar_label: "All functions"
  This document describes the SQL language.
 :::
 
+This page provides a reference of all Druid SQL functions in alphabetical order. The **Learn More** at the end of each function section provides further documentation.
 
-This page provides a reference of all Druid SQL functions in alphabetical order.
-Click the linked function type for documentation on a particular function.
+The examples for each function uses the following datasets that come included with Apache Druid:
+* `flight-carriers`: `FlightCarrierOnTime (1 month)` 
+* `trips_xaa`: `NYC Taxi cabs (3 files)`
```

Comment:
```suggestion
* `FlightCarrierOnTime (1 month)` abbreviated as `flight-carriers`
* `NYC Taxi cabs (3 files)` abbreviated as `taxi-trips`
```
Lead with the examples as named in Druid and follow with the abbreviation.
Recommend using a standard datasource naming convention. (in this case a dash)
Also, use English in example names where possible